pyproject.toml

```diff
@@ -0,0 +1,117 @@
+[build-system]
+
+requires = [
+    "setuptools>=45.0",
+    "setuptools_scm[toml]>=6.2",
+    "wheel>=0.38.1",
+]
+
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+packages = ["pyscal"]
```

Comment:
Is this needed because this repository isn't `src/` based?

Reply:
yes. I was tempted to move `pyscal` folder inside `src` But decided to use this instead. Any other benefit of using `src/` based?

Reply:
Good point. I'll adopt this on next PR :+1: